addons/mail/static/src/components/composer/composer_tests.js

```diff
@@ -1842,6 +1842,22 @@ QUnit.test('remove an uploading attachment aborts upload', async function (asser
     });
 });
 
+QUnit.test('basic rendering when thread has no name', async function (assert) {
```

Comment:
name is to abstract, should clearly tell it's about sending message composer that has title telling it's addressing to followers of this document.